From a5cb55a919dd1b07a4c3bb3a1e5dbbe7b361b7f4 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 15 Feb 2018 18:31:47 -0800
Subject: [PATCH] TRITON-139 Triton's config-agents should poll at 120s
 interval as a stopgap (update default pollInterval) Reviewed by: Orlando
 Vazquez <orlando@joyent.com> Approved by: Orlando Vazquez
 <orlando@joyent.com>

---
 agent.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/agent.js b/agent.js
index 0996195..25736f8 100644
--- a/agent.js
+++ b/agent.js
@@ -68,7 +68,7 @@ if (configPath) {
 	config = {
 		instances: [],
 		logLevel: 'info',
-		pollInterval: 60000
+		pollInterval: 120000
 	};
 }
 
-- 
2.21.0

